Externals/mGBA/mgba.vcxproj

```diff
@@ -26,123 +26,95 @@
   </ItemDefinitionGroup>
   <ItemGroup>
     <ClCompile Include="mgba\src\core\bitmap-cache.c" />
-    <ClCompile Include="mgba\src\core\cache-set.c">
-      <ObjectFileName>$(IntDir)/src/core/cache-set.c.obj</ObjectFileName>
-    </ClCompile>
-    <ClCompile Include="mgba\src\core\cheats.c">
-      <ObjectFileName>$(IntDir)/src/core/cheats.c.obj</ObjectFileName>
-    </ClCompile>
+    <ClCompile Include="mgba\src\core\cache-set.c" />
+    <ClCompile Include="mgba\src\core\cheats.c" />
     <ClCompile Include="mgba\src\core\config.c" />
-    <ClCompile Include="mgba\src\core\core.c">
-      <ObjectFileName>$(IntDir)/src/core/core.c.obj</ObjectFileName>
-    </ClCompile>
+    <ClCompile Include="mgba\src\core\core.c" />
     <ClCompile Include="mgba\src\core\directories.c" />
-    <ClCompile Include="mgba\src\core\input.c">
-      <ObjectFileName>$(IntDir)/src/core/input.c.obj</ObjectFileName>
-    </ClCompile>
+    <ClCompile Include="mgba\src\core\input.c" />
     <ClCompile Include="mgba\src\core\interface.c" />
     <ClCompile Include="mgba\src\core\library.c" />
-    <ClCompile Include="mgba\src\core\lockstep.c">
-      <ObjectFileName>$(IntDir)/src/core/lockstep.c.obj</ObjectFileName>
-    </ClCompile>
+    <ClCompile Include="mgba\src\core\lockstep.c" />
     <ClCompile Include="mgba\src\core\log.c" />
     <ClCompile Include="mgba\src\core\map-cache.c" />
     <ClCompile Include="mgba\src\core\mem-search.c" />
     <ClCompile Include="mgba\src\core\rewind.c" />
     <ClCompile Include="mgba\src\core\scripting.c" />
-    <ClCompile Include="mgba\src\core\serialize.c">
-      <ObjectFileName>$(IntDir)/src/core/serialize.c.obj</ObjectFileName>
-    </ClCompile>
+    <ClCompile Include="mgba\src\core\serialize.c" />
     <ClCompile Include="mgba\src\core\sync.c" />
     <ClCompile Include="mgba\src\core\thread.c" />
     <ClCompile Include="mgba\src\core\tile-cache.c" />
     <ClCompile Include="mgba\src\core\timing.c" />
-    <ClCompile Include="mgba\src\sm83\decoder.c">
-      <ObjectFileName>$(IntDir)/src/sm83/decoder.c.obj</ObjectFileName>
-    </ClCompile>
+    <ClCompile Include="mgba\src\sm83\decoder.c" />
     <ClCompile Include="mgba\src\sm83\isa-sm83.c" />
     <ClCompile Include="mgba\src\sm83\sm83.c" />
-    <ClCompile Include="mgba\src\gb\audio.c">
-      <ObjectFileName>$(IntDir)/src/gb/audio.c.obj</ObjectFileName>
-    </ClCompile>
+    <ClCompile Include="mgba\src\gb\audio.c" />
     <ClCompile Include="mgba\src\gb\cheats.c">
-      <ObjectFileName>$(IntDir)/src/gb/cheats.c.obj</ObjectFileName>
+      <ObjectFileName>$(IntDir)obj2\</ObjectFileName>
```

Comment:
Could we call those `gb` and `gba`?

Reply:
I decided not to because the (object) files aren't actually split up that way (because there are sometimes duplicate names outside of gb/gba dirs)